src/index.js

```diff
@@ -3,23 +3,63 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
-import App from './Containers/App/App';
+import { ThemeProvider } from '@material-ui/styles';
+import { createMuiTheme } from '@material-ui/core/styles';
+import App from './containers/App/App';
 import createStore from './createStore';
 import * as serviceWorker from './serviceWorker';
 import './assets/css/normalize.css';
 import './assets/css/fonts.css';
 import './assets/css/main.css';
+import './assets/css/var.css';
 
 // const initStore = {};
 const store = createStore;
 
+const theme = createMuiTheme({
```

Comment:
we have theme.json

Reply:
that theme.json is not working for now. And that part will be resolved on themeProveider branch soon :)